src/lib/datepicker/datepicker.ts

```diff
@@ -167,6 +167,12 @@ export class MatDatepicker<D> implements OnDestroy {
    */
   @Output() selectedChanged = new EventEmitter<D>();
 
+  /** Emits when the datepicker has been opened. */
+  @Output('opened') openedStream: EventEmitter<void> = new EventEmitter<void>();
```

Comment:
@amcdnl @jelbourn opinions on what we call these? I think select calls them `onOpen` and `onClose` (for both property and output name)

Reply:
@mmalerba I based the naming on https://github.com/angular/material2/pull/7160 where `onOpen` and `onClose` were changed to `opened` and `closed`.

Reply:
ah ok, can we just change the `closed` property to `closedStream` then so that it matches `openedStream`?

Reply:
Done.